### PR TITLE
Add dark mode support.

### LINKED ISSUE #16: Dark mode
I think a dark mode would be a valuable addition to the app, especially since it's going to be mainly used at night. I've never made an android app before but I changed some variables around after installing android studio which resulted with [this](https://gist.github.com/alvierahman90/e679934770d03f81e3db63e1c9987a12) which looks like this:

Preferably the app will be able to switch between dark and light modes as android changes between dark and light mode. 

I will try to figure out how to do it myself and submit a pull request but, as I said, I've never done android development before so no guarantees.

### PR DESCRIPTION
**Pull request by martiandolphin**
closes #16

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.DayNight.DarkActionBar">
         <!-- Customize your theme here. -->
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
+        <item name="android:background">@color/colorBackground</item>
```
Comment (vmiklos) [inline app/src/main/res/values/styles.xml:9]: You could just omit this line, that way you get sensible default background colors: purple and white for the AppBar/rest in light mode, and almost-black gray in dark mode. I'll fix that in a follow-up commit.

**Comment by vmiklos** [comment]
Thanks, this looks nice! :-)

See inline commit as to how we could fix the white-on-white situation for the ActionBar.

File: app/src/main/res/values/styles.xml (hunk 1 above)
Comment (martiandolphin) [reply app/src/main/res/values/styles.xml:9]: The almost-black gray is better than blinding white for sure but I'd quite like a pure black option as well for dark mode. What's the best way to add that? Maybe a checkbox in the options menu?

Comment (vmiklos) [reply app/src/main/res/values/styles.xml:9]: I don't have a strong opinion on this, so if you find a way to have completely black background, but keep the light case unchanged, I would accept that. An option for this is probably an overkill :)

**Pull request merged**